packages/mobile/views/dashboard/drawers/profile/views/settings/views/networkConfiguration/views/NetworkConfigurationAddUpdateNodeView.svelte

```diff
@@ -16,22 +14,20 @@
     let nodeConfigurationForm: NodeConfigurationForm
     let isBusy = false
 
-    async function handleSubmit(): Promise<void> {
+    async function onSubmit(): Promise<void> {
```

Comment:
```suggestion
    async function onSubmitClick(): Promise<void> {
```

Reply:
I don't think we should change it, since `onSubmit` can also be triggered by pushing an enter button.